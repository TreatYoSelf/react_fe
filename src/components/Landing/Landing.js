import React from 'react';
import { StyleSheet, Text, Image, View, Button } from 'react-native';

export default function Landing(props) {
    return (
        <View>
            <Image source={require("../../../assets/logo.png")} style={{ width: 200, height: 200 }} />
            <Text>An app where you can treat yourself to some personal time, guilt free.</Text>
            <Text style={styles.header}>Sign In With Google</Text>
            <Button title="Sign in with Google" onPress={() => props.signIn()} />
        </View>
    );
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: "#fff",
        alignItems: "center",
        justifyContent: "center"
    },
    header: {
        fontSize: 25
    },
    image: {
        marginTop: 15,
        width: 150,
        height: 150,
        borderColor: "rgba(0,0,0,0.2)",
        borderWidth: 3,
        borderRadius: 150
    }
})
// export default Landing;