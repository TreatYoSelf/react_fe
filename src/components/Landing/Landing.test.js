import React from 'react';
import renderer from 'react-test-renderer';

import Landing from './Landing';

describe('<Landing />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Landing />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});