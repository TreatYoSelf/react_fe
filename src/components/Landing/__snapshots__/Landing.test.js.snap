// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders correctly 1`] = `
<View>
  <Image
    source={1}
    style={
      Object {
        "height": 200,
        "width": 200,
      }
    }
  />
  <Text>
    An app where you can treat yourself to some personal time, guilt free.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 25,
      }
    }
  >
    Sign In With Google
  </Text>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Sign in with Google
      </Text>
    </View>
  </View>
</View>
`;
